starter_code/src/movies.js

```diff
@@ -1,15 +1,61 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(array) {
+    var movies = array.map(function (movie) {
+        let durationArray = movie.duration.split(' ');
+        let firstNumber = parseInt(durationArray[0].slice(0, 1)) * 60;
+        let secondNumber = parseInt(durationArray[1].slice(0,2));
+        let output = firstNumber + secondNumber;
+        movie.duration = output;
+        return movie;
+    });
+}
```

Comment:
I made couple corrections so we can relate movie.duration with THE movie itself:
```
function turnHoursToMinutes(array) {
    return array.map(function (movie) {
      
        let durationArray = movie.duration.split(" ");
        let firstNumber = parseInt(durationArray[0].slice(0, 1)) * 60;
        let secondNumber = parseInt(durationArray[1].slice(0,2));
        let output = firstNumber + secondNumber;
        // console.log("output", output)
        return Object.assign({}, movie, {duration: output});
    });
}
console.log(turnHoursToMinutes(movies)) 
```